nixos/modules/security/hidepid.nix

```diff
@@ -17,10 +17,9 @@ in
         the process information of other users.
       '';
       };
-      procGID = mkOption {
-        type = types.nullOr types.int;
-        default = null;
-        example = 1337;
+      procGroup = mkOption {
+        type = types.nullOr types.str;
+        example = "proc";
```

Comment:
Oh, I see now -- we need to have GID in mount options. What about allocating a dedicated group in `ids.nix` and using it instead of `procGroup`?